odrive_node/msg/ODriveStatus.msg

```diff
@@ -1,3 +1,5 @@
+std_msgs/Header header
```

Comment:
Since we're not using any of the other fields in `Header`, is there any downside to just using `time timestamp` directly?